system_test/aest_commands_SUITE.erl

```diff
@@ -0,0 +1,84 @@
+-module(aest_commands_SUITE).
+
+%=== EXPORTS ===================================================================
+
+% Common Test exports
+-export([all/0]).
+-export([init_per_testcase/2]).
+-export([end_per_testcase/2]).
+
+% Test cases
+-export([epoch_commands/1]).
+
+-import(aest_nodes, [
+    setup_nodes/2,
+    start_node/2,
+    wait_for_value/4
+]).
+
+%=== INCLUDES ==================================================================
+
+-include_lib("eunit/include/eunit.hrl").
+
+%=== MACROS ====================================================================
+
+-define(MINING_TIMEOUT, 2000).
+-define(STARTUP_TIMEOUT, 5000).
+
+-define(NODE1, #{
+    name    => node1,
+    peers   => [],
+    backend => aest_docker,
+    source  => {pull, "aeternity/epoch:local"}
+}).
+
+-define(EPOCH_VERSION_OUTPUT_RX, "Installed versions:[\r\n]*\\*[ \t]*[0-9.]*[ \t]*permanent[\r\n]*").
+-define(EPOCH_CHECK_OUTPUT_RX, "OK[\r\n]*").
+-define(EPOCH_PID_OUTPUT_RX, "1[\r\n]*").
+-define(EPOCH_PING_OUTPUT_RX, "pong[\r\n]*").
+
+
+%=== COMMON TEST FUNCTIONS =====================================================
+
+all() -> [
+    epoch_commands
+].
+
+init_per_testcase(_TC, Config) ->
+    aest_nodes:ct_setup(Config).
+
+end_per_testcase(_TC, Config) ->
+    aest_nodes:ct_cleanup(Config).
+
+
+%=== TEST CASES ================================================================
+
+epoch_commands(Cfg) ->
+    setup_nodes([?NODE1], Cfg),
+    start_node(node1, Cfg),
+    wait_for_value({height, 0}, [node1], ?STARTUP_TIMEOUT, Cfg),
+    wait_for_value({height, 10}, [node1], ?MINING_TIMEOUT * 10, Cfg),
+    {0, Output1} = aest_nodes:run_cmd_in_node_dir(node1, ["bin/epoch", "versions"], Cfg),
+    ?assertMatch({match, _}, re:run(Output1, ?EPOCH_VERSION_OUTPUT_RX)),
```

Comment:
I think having the output regex defined somewhere else makes the test harder to read. It's better if all the test data is inside the test itself. This comment applies to all macros used in the asserts below as well.